src/types.js

```diff
@@ -1,12 +1,72 @@
 export const TYPES = {
   tos: {
-    name: 'Terms of service',
+    name: 'Terms of Service',
     fileName: 'terms_of_service'
   },
+  website_tos: {
```

Comment:
In favor of merging with `tos`, unless I misunderstand and there is a difference.

Reply:
It's used by for instance Apple to define the ToS of the www.apple.com website, as opposed to their iChat ToS, Game Center ToS, iCloud ToS, and iTunes ToS. Maybe the right thing to do here is to define "the Apple website", iChat, Apple Game Center, iCloud and iTunes all as separate services. It's probably inaccurate to define "Apple" as a single service.

Reply:
That way, we can have "the `tos` of www.apple.com" instead of "the `website_tos` of Apple".

Reply:
> Maybe the right thing to do here is to define "the Apple website", iChat, Apple Game Center, iCloud and iTunes all as separate services.

Yes, strong +1.
This is also what I suggested and we finally decided in #77: Facebook, Facebook Ads and Facebook Payments should all be separate services. This suggests there might be a use for modeling the service _provider_ in the end, but right now let's declare each _service_ independently 🙂